library_management_cookiecutter/templates/base.html

```diff
@@ -1,31 +1,88 @@
+<!DOCTYPE html>
+<html>
 
-{% load static %}
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css" rel="stylesheet" />
```

Comment:
create  base.html file for common functionalities which is applicable for all html files.